spec/java_buildpack/framework/new_relic_agent_spec.rb

```diff
@@ -59,6 +59,7 @@
       expect(java_opts).to include("-javaagent:$PWD/.java-buildpack/new_relic_agent/new_relic_agent-#{version}.jar")
       expect(java_opts).to include('-Dnewrelic.home=$PWD/.java-buildpack/new_relic_agent')
       expect(java_opts).to include('-Dnewrelic.config.license_key=test-license-key')
+      expect(java_opts).to include('-Dnewrelic.config.agent_enabled=true')
```

Comment:
Feel like we should have test(s) to check that this option isn't set if new_relic_agent_enabled is missing, blank, etc.